actor/deadletter.go

```diff
@@ -17,7 +17,11 @@ func NewDeadLetter(actorSystem *ActorSystem) *deadLetterProcess {
 	actorSystem.ProcessRegistry.Add(dp, "deadletter")
 	_ = actorSystem.EventStream.Subscribe(func(msg interface{}) {
 		if deadLetter, ok := msg.(*DeadLetterEvent); ok {
-			plog.Debug("[DeadLetter]", log.Stringer("pid", deadLetter.PID), log.Message(deadLetter.Message), log.Stringer("sender", deadLetter.Sender))
+			plog.Debug("[DeadLetter]", log.Stringer("pid", deadLetter.PID), log.TypeOf("msg", deadLetter.Message), log.Stringer("sender", deadLetter.Sender))
+			// send back a response instead of timeout.
+			if deadLetter.Sender != nil {
+				actorSystem.Root.Send(deadLetter.Sender, &DeadLetterResponse{})
```

Comment:
👍🏻 good to see that we have this part in Go now.

Reply:
@cupen should we also add some data to `DeadLetterResponse` like PID or message so that the sender has some context to understand why it got this response.

Reply:
@bran-san There is only one reason that the sender get a `DeadLetterResponse`, it request a unreachable actor.